### PR TITLE
Enable using WPIMULT in ACTIONX.

### PR DESCRIPTION
**Pull request by atgeirr**
Without this, I have a field case that fails.

Behaviour may be wrong in corner cases that have the action triggering at the same time as WPIMULT keywords are present in the deck, but I thing it is probably not worth it to address.

### CODE REVIEW

**Comment by atgeirr** [comment]
jenkins build this please

**Comment by blattms** [comment]
Should we add a test?

**Comment by OPMUSER** [comment]
If you mean an opm-test type of test, then we have one already, but it does not test

> Behaviour may be wrong in corner cases that have the action triggering at the same time as WPIMULT keywords are present in the deck, but I thing it is probably not worth it to address.

I think we should have a test for these conditions so we know what happens under these circumstances, whether or not we account for this in another question though.

Let me know if need the test.

File: src/opm/input/eclipse/Schedule/Schedule.cpp [hunk 1]
```diff
@@ -1293,6 +1293,7 @@ File {} line {}.)", pattern, location.keyword, location.filename, location.linen
         const std::string prefix = "| "; /* logger prefix string */
         this->snapshots.resize(reportStep + 1);
         auto& input_block = this->m_sched_deck[reportStep];
+        std::unordered_map<std::string, double> wellpi_global_factor;
```
Comment (GitPaean) [inline src/opm/input/eclipse/Schedule/Schedule.cpp:1296]: It is much appreciated if you can change the name `wellpi_global_factor` to be `wpimult_global_factor`. It should have been done in the previous PR while it got merged first.

**Comment by GitPaean** [comment]
Regarding tests, maybe a test in `ScheduleTests` is the easiest?

File: src/opm/input/eclipse/Schedule/Schedule.cpp (hunk 1 above)
Comment (atgeirr) [reply src/opm/input/eclipse/Schedule/Schedule.cpp:1296]: Good point, did not notice but will change it.

**Comment by atgeirr** [comment]
Yes I think we should add a test. Simply adding
```
ACTIONX
REDP12 1 /
DAY >= 11 AND /
MNTH >= JAN AND /
YEAR >= 2015 /
/
WPIMULT
'PRODU12'    0.2 /
/
ENDACTIO
```
to SPE9_CP.DATA should error out with the current master. I guess a tiny 1D testcase should be sufficient even.

**Comment by atgeirr** [comment]
jenkins build this please

**Comment by OPMUSER** [comment]
In opm-test, there is a whole series of ACTIONX tests so we should generate a test based on ACTIONX_WPIMULT, ACTIONX_WPIMULT-01. The current test previously worked when tested.

**Comment by bska** [comment]
> we should generate a test based on [ACTIONX_WPIMULT](https://github.com/OPM/opm-tests/blob/295eca9a1faed4e0614919ac1718b79140ee007b/actionx/ACTIONX_WPIMULT.DATA)

For what it's worth, I altered that simulation model to also report the connection transmissiblity factors of two selected well connections (PR OPM/opm-tests#787) and ran the current PR locally.  I got the figure below which demonstrates that this PR restores the earlier ability.  I am going to merge this change now and create a regression test based on the new version of the simulation model.

---

ACTIONX_WPIMULT_CTFAC
